pkg/proxy/proxy.go

```diff
@@ -161,12 +161,45 @@ func filterList(rels []*release.Release) []*release.Release {
 	return uniq
 }
 
+func getStatuses(statusQuery string) []release.Status_Code {
```

Comment:
What will happen in the query is invalid? Will we return an empty list? An error?

Reply:
An empty list. I think that's the most logic thing to do. Being literal any app will comply with that status. Note that the "error" will be logged.